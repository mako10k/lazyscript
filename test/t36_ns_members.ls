!{
  ns <- !nsnew0;
  !nsdefv ~ns .a 1;
  !nsdefv ~ns .b 2;
  !nsdefv ~ns .aa 3;
  !nsdefv ~ns .ab 4;
  ~~println (~~to_str ((~prelude nsMembers) ~ns));
};
