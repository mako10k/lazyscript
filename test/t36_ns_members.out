[.a, .b, aa, ab, 10, 2, Bar, Foo]
()
